trolldb/tests/tests_api/test_api.py

```diff
@@ -38,7 +38,15 @@ def single_query_is_correct(key: str, value: str | datetime) -> bool:
     )
 
 
-def query_results_are_correct(keys: list[str], values_list: list[list[str | datetime]]) -> bool:
+def make_query_string(keys: list[str], values_list: list[list[str] | datetime]) -> str:
+    """Makes a single query string for all the given queries."""
+    query_buffer = []
+    for key, value_list in zip(keys, values_list, strict=True):
+        query_buffer += [f"{key}={value}" for value in value_list]
+    return "&".join(query_buffer)
+
```

Comment:
Not a big deal, but the way we usually do in pytroll is to have the dependent functions after the function that uses them.
Eg:
```
def main():
	func1()
	func2()

def func1():
	...

def func2():
	...
```

Helps with readability.

Reply:
I realise though I probably didn't take the time to reorder in cli.py, sorry.

Reply:
I just reordered the functions in test scripts. I did not the fixtures though!